sci-biology/arb/arb-6.0.6.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Tools for DNA/RNA sequence database handling and phylogenetic analysis"
+HOMEPAGE="http://www.arb-home.de/"
+SRC_URI="http://download.arb-home.de/release/${P}/${P}-source.tgz -> ${P}.tgz"
+
+SLOT="0"
+LICENSE="arb"
+IUSE="debug +opengl test"
+KEYWORDS="~amd64 ~x86"
+
+CDEPEND="app-text/sablotron
+	media-libs/libpng:=
+	media-libs/tiff:=
+	www-client/lynx
+	x11-libs/libXaw
+	x11-libs/libXpm
+	x11-libs/motif:0
+	opengl? (
+		media-libs/glew:=
+		media-libs/freeglut
+		|| (
+			media-libs/mesa[motif]
+			( media-libs/mesa x11-libs/libGLw ) ) )"
+DEPEND="${CDEPEND}
+	sys-process/time
+	x11-misc/makedepend"
+RDEPEND="${CDEPEND}
+	sci-visualization/gnuplot"
+
+# Almost half of tests are broken with debug
+RESTRICT="debug? ( test )"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-6.0.6-glapi.patch
+	"${FILESDIR}"/${PN}-6.0.6-tc-flags.patch
+	"${FILESDIR}"/${PN}-6.0.6-arb_install.patch
+)
+
+src_unpack() {
+	default
+	mv arbsrc* ${P}
```

Comment:
missing `die ||`. There are missing dies all over the place. Could you please add them? Thanks.

Reply:
No problem.  I was just using `arb-5.3.ebuild` as a template but I'll clean it up.